--- chat-superpaper/run
+++ chat-superpaper/run
@@ -6,4 +6,7 @@ if [ "$DEBUG_NODE" == "true" ]; then
     NODE_PARAMS="--inspect=0.0.0.0:30100"
 fi

+source /etc/superpaper/env.sh
+export LISTEN_ADDRESS=127.0.0.1
+
 exec /sbin/setuser www-data /usr/bin/node $NODE_PARAMS /superpaper/services/chat/app.js >> /var/log/superpaper/chat.log 2>&1
--- clsi-superpaper/run
+++ clsi-superpaper/run
@@ -15,4 +15,7 @@ if [ -e '/var/run/docker.sock' ]; then
   usermod -aG dockeronhost www-data
 fi

+source /etc/superpaper/env.sh
+export LISTEN_ADDRESS=127.0.0.1
+
 exec /sbin/setuser www-data /usr/bin/node $NODE_PARAMS /superpaper/services/clsi/app.js >> /var/log/superpaper/clsi.log 2>&1
--- contacts-superpaper/run
+++ contacts-superpaper/run
@@ -6,4 +6,7 @@ if [ "$DEBUG_NODE" == "true" ]; then
     NODE_PARAMS="--inspect=0.0.0.0:30360"
 fi

+source /etc/superpaper/env.sh
+export LISTEN_ADDRESS=127.0.0.1
+
 exec /sbin/setuser www-data /usr/bin/node $NODE_PARAMS /superpaper/services/contacts/app.js >> /var/log/superpaper/contacts.log 2>&1
--- docstore-superpaper/run
+++ docstore-superpaper/run
@@ -6,4 +6,7 @@ if [ "$DEBUG_NODE" == "true" ]; then
     NODE_PARAMS="--inspect=0.0.0.0:30160"
 fi

+source /etc/superpaper/env.sh
+export LISTEN_ADDRESS=127.0.0.1
+
 exec /sbin/setuser www-data /usr/bin/node $NODE_PARAMS /superpaper/services/docstore/app.js >> /var/log/superpaper/docstore.log 2>&1
--- document-updater-superpaper/run
+++ document-updater-superpaper/run
@@ -6,4 +6,7 @@ if [ "$DEBUG_NODE" == "true" ]; then
     NODE_PARAMS="--inspect=0.0.0.0:30030"
 fi

+source /etc/superpaper/env.sh
+export LISTEN_ADDRESS=127.0.0.1
+
 exec /sbin/setuser www-data /usr/bin/node $NODE_PARAMS /superpaper/services/document-updater/app.js >> /var/log/superpaper/document-updater.log 2>&1
--- filestore-superpaper/run
+++ filestore-superpaper/run
@@ -1,2 +1,6 @@
 #!/bin/bash
+
+source /etc/superpaper/env.sh
+export LISTEN_ADDRESS=127.0.0.1
+
 exec /sbin/setuser www-data /usr/bin/node /superpaper/services/filestore/app.js >> /var/log/superpaper/filestore.log 2>&1
--- notifications-superpaper/run
+++ notifications-superpaper/run
@@ -6,4 +6,7 @@ if [ "$DEBUG_NODE" == "true" ]; then
     NODE_PARAMS="--inspect=0.0.0.0:30420"
 fi

+source /etc/superpaper/env.sh
+export LISTEN_ADDRESS=127.0.0.1
+
 exec /sbin/setuser www-data /usr/bin/node $NODE_PARAMS /superpaper/services/notifications/app.js >> /var/log/superpaper/notifications.log 2>&1
--- project-history-superpaper/run
+++ project-history-superpaper/run
@@ -6,4 +6,7 @@ if [ "$DEBUG_NODE" == "true" ]; then
     NODE_PARAMS="--inspect=0.0.0.0:30540"
 fi

+source /etc/superpaper/env.sh
+export LISTEN_ADDRESS=127.0.0.1
+
 exec /sbin/setuser www-data /usr/bin/node $NODE_PARAMS /superpaper/services/project-history/app.js >> /var/log/superpaper/project-history.log 2>&1
--- real-time-superpaper/run
+++ real-time-superpaper/run
@@ -1,2 +1,6 @@
 #!/bin/bash
+
+source /etc/superpaper/env.sh
+export LISTEN_ADDRESS=127.0.0.1
+
 exec /sbin/setuser www-data /usr/bin/node /superpaper/services/real-time/app.js >> /var/log/superpaper/real-time.log 2>&1
--- spelling-superpaper/run
+++ spelling-superpaper/run
@@ -6,4 +6,7 @@ if [ "$DEBUG_NODE" == "true" ]; then
     NODE_PARAMS="--inspect=0.0.0.0:30050"
 fi

+source /etc/superpaper/env.sh
+export LISTEN_ADDRESS=127.0.0.1
+
 exec /sbin/setuser www-data /usr/bin/node $NODE_PARAMS /superpaper/services/spelling/app.js >> /var/log/superpaper/spelling.log 2>&1
--- web-api-superpaper/run
+++ web-api-superpaper/run
@@ -6,6 +6,7 @@ if [ "$DEBUG_NODE" == "true" ]; then
     NODE_PARAMS="--inspect=0.0.0.0:30000"
 fi

+source /etc/superpaper/env.sh
 export LISTEN_ADDRESS=0.0.0.0
 export ENABLED_SERVICES="api"
 export METRICS_APP_NAME="web-api"
--- web-superpaper/run
+++ web-superpaper/run
@@ -6,6 +6,8 @@ if [ "$DEBUG_NODE" == "true" ]; then
     NODE_PARAMS="--inspect=0.0.0.0:40000"
 fi

+source /etc/superpaper/env.sh
+export LISTEN_ADDRESS=127.0.0.1
 export ENABLED_SERVICES="web"
 export WEB_PORT="4000"
